docker-compose.yml

```diff
@@ -28,15 +28,6 @@ services:
       - net_db_bridge_senderforeign
     restart: unless-stopped
     volumes: ['~/bridge_data/redis:/data']
-  bridge:
```

Comment:
Should we update https://github.com/poanetwork/token-bridge/tree/e9588a44e4ecd433fe1b7fb461a33315cdbbb889#rollback-the-last-processed-block-in-redis and https://github.com/poanetwork/token-bridge/tree/e9588a44e4ecd433fe1b7fb461a33315cdbbb889#native-to-erc20-mode-testing since they use `bridge` service?